surya.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "shoya0x00",
+			"oem": "Poco",
+			"device": "POCO X3 (NFC)",
+            "filename": "crDroidAndroid-16.0-20250905-surya-v12.0.zip",
+            "download": "https://sourceforge.net/projects/crdroid/files/surya/12.x/crDroidAndroid-16.0-20250905-surya-v12.0.zip/download",
+            "timestamp": 1757091852,
+            "md5": "1c292513e4fe0fd9eb0dba376ff2f50f",
+            "sha256": "7a66e300e0ce2cd7ec6dfc9af7e04cf3b9ccaff116dc0794c693b9b22c882391",
+            "size": 1787557384,
+            "version": "12.0",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-android-15-0-surya-crdroid-v11-stable-official.4700889/",
```

Comment:
update forum links
- features: https://github.com/crdroidandroid/crdroid_features/blob/16.0/README.mkdn
- rom: https://crdroid.net/surya/12
- recovery (guessing can point to download page as it will be listed there or remove it)
- support (should be one link only): https://crdroid.net/surya/12/support (will list telegram page there automatically as with 11)
- maybe add link to install instructions page also? : https://crdroid.net/surya/12/install